### PR TITLE
Added Uploading feature

### PR DESCRIPTION
**Pull request by prakkhar03**
Added uploading feature in which user can add the recipe its photo and video

### CODE REVIEW

File: cooknookapp/models.py
Comment (urmarc443) [inline cooknookapp/models.py:1]: Should upload() be a function instead of a class? What state is needed for the upload?

Comment (prakkhar03) [reply cooknookapp/models.py:1]: If the upload process involves multiple steps (e.g., preparing, validating, uploading in chunks, retrying on failure), a class is better suited. You may want to store the state across methods, making a class useful.

Comment (urmarc443) [reply cooknookapp/models.py:1]: That makes sense, I appreciate the concise and prompt reply.

Comment (prakkhar03) [reply cooknookapp/models.py:1]: It can be further more modified 
it also help in easy displaying of data

**Comment by morganog** [comment]
Thanks for this addition! I am going to go ahead and merge this.

**Pull request merged**